### ISSUE TITLE
Select cell selecting its menu

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#122320]
Testing #122254

* open a GHI issue notebook
* select the first code cell
* open the cell overflow menu of the second cell
* :bug: the first cell is still selected

**Comment by roblourens** [comment]
So when I open this menu, the second cell should get selected?

![image](https://user-images.githubusercontent.com/323878/116260812-e3ab0380-a72b-11eb-96c4-995852667462.png)

**Comment by jrieken** [comment]
> So when I open this menu, the second cell should get selected?

Yeah, I would say so. At least for certain commands. My use-case was: select collapse input, see and remember its shortcut, trigger the shortcut, another cell collapses...
